src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -138,26 +145,47 @@
          to one of its options).  Simply appending the traits would break code coverage. -->
     <PropertyGroup>
       <TestCommandLine>$(TestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</TestCommandLine>
+      <BareTestCommandLine>$(BareTestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</BareTestCommandLine>
```

Comment:
Sorry I missed the one difference here but I do still think we should fix this up above and make them identical, I will add more comments above.